### ISSUE TITLE
Missing MacOS and Linux wheels for Python 3.11

### ISSUE THREAD

**Issue by mjpieters** [cvxopt/cvxopt-wheels#10]
I noticed that [windows wheels have been uploaded](https://pypi.org/project/cvxopt/#files). Can we have wheels for MacOS and Linux too?

In addition, the MacOS wheels for M1 machines are needed (ending in `-macosx_11_0_arm64.whl`). You could consider using https://github.com/pypa/cibuildwheel to do the building (set [`CIBW_ARCHS_MACOS: x86_64 arm64 universal2`](https://cibuildwheel.readthedocs.io/en/stable/options/#archs) to ensure the rigth MacOS wheels are generated).

**Comment by martinandersen** [comment]
Yes, updated macOS and Linux wheels are on our to-do list. In principle, this should be effortless, but for some reason, the deployment step of the Linux build on Travis CI has started failing and needs attention.

**Comment by burk** [comment]
I gave it a go here: https://github.com/cvxopt/cvxopt-wheels/pull/11

**Comment by mjpieters** [comment]
This has long since been resolved. Thanks again!
